cmd/news/news.go

```diff
@@ -0,0 +1,78 @@
+package news
+
+/*
+Practise go by creating a function that retrieves an the information from the following BBC News xml link and gives a list of all the current headlines.
+http://feeds.bbci.co.uk/news/technology/rss.xml
+*/
+
+import (
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	url = "http://feeds.bbci.co.uk/news/technology/rss.xml"
+)
+
+// The top level structure
+type RSS struct {
+	Item []Item `xml:"channel>item"`
+}
+
+// The structure of the information we wish to retrieve
+type Item struct {
+	Title       string `xml:"title"`
+	Description string `xml:"description"`
+	Link        string `xml:"link"`
+	Guid        string `xml:"guid"`
+	PubDate     string `xml:"pubDate"`
+}
+
+func CreateCommand() *cobra.Command {
```

Comment:
add a `--format` flag that will take a value of `json` or `table`. When the flag is set to json, output the list of items in json e.g. 
``` json
{
	"items": [{
			"title": "some title",
			"url": "http://path/to/article"
		},
		{
			"title": "another title",
			"url": "http://path/to/article"
		}
	]
}
```